helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -93,6 +93,13 @@ internal class ListLicensesCommand : CommandWithHelp() {
     )
     private var noLicenseTexts: Boolean = false
 
+    @Parameter(
+        names = ["--apply-license-finding-curations"],
+        required = false,
+        order = PARAMETER_ORDER_OPTIONAL
+    )
```

Comment:
Please add a description, also see https://github.com/heremaps/oss-review-toolkit/issues/1817.

Reply:
Are you fine to defer description to #1817?
..for consistentcy, priorization and development speed.

Reply:
Added all the missing descriptions to this classes' parameters.